metrics.md

```diff
@@ -0,0 +1,47 @@
+|gluster_volume_total_bytes| gluster_volume_total_bytes{volume="xxx"}  104857600	|
+|gluster_volume_used_bytes| gluster_volume_used_bytes{volume="xxx"}  4857600	|
+|gluster_volume_available_bytes|	gluster_volume_available_bytes{volume="xxx"} 9856780	|
+
+#### Brick Metrics:
+|Gluster Promethus schema| example
+|--------|-----------|
+| gluster_brick_inodes_used |	gluster_brick_inodes_used{volume="xxx", "brick_path"="/mnt/xxx", nodeid="xxxxx"} 100	|
+| gluster_brick_inodes_total |	gluster_brick_inodes_total{volume="xxx", "brick_path"="/mnt/xxx", nodeid="xxxxx"} 200 |	
+| gluster_brick_inodes_available | gluster_brick_inodes_available{volume="xxx", "brick"="/mnt/xxx", nodeid="xxxxx"} 100 |
+| gluster_brick_size_bytes | gluster_brick_size_bytes{volume="xxx", brick_path="xxxx", nodeid="xxx"} 1048576	|
+| gluster_brick_used_bytes|	gluster_brick_used_bytes{volume="xxx", brick_path="xxxx", nodeid="xxx"} 1048	|
+| gluster_brick_available_bytes |	gluster_brick_available_bytes{volume="xxx", brick_path="xxx", nodeid="xxx"} 1047576	|
+| gluster_disk_read_ios | gluster_disk_read_ios{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_read_merges | gluster_disk_read_merges{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_read_sectors | gluster_disk_read_sectors{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_read_ticks | gluster_disk_read_ticks{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_write_ios | gluster_disk_write_ios{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0
+|   gluster_disk_write_merges | gluster_disk_write_merges{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_write_sectors | gluster_disk_write_sectors {brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_write_ticks | gluster_disk_write_ticks {brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_inflight | gluster_disk_inflight{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|   gluster_disk_total_ticks | gluster_disk_total_ticks{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+|  gluster_disk_time_inqueue | gluster_disk_time_inqueue{brick_path="xxx",device="xxx",disk="vda",host="xxx",id="xxx",instance="xxx:8080",subvolume="xxx",volume="xxx"} 0 |
+
+#### Gluster process Metrics: 
```

Comment:
@vredara How do we find if a brick is online or not?

Reply:
we can add metrics for this.

1. gluster_brick_status {brick_path="xxx", vol="xxx", peerid="xx"} 1 (1 for up, 0 for down)
2. gluster_vol_numbricks {vol="xxx"} <number_of_bricks>

we need to have prometheus rule like sum(gluster_brick_status{vol="xxx"}) == gluster_vol_numbricks{vol="xxx"} . if the sum doesnt tally we have brick which is down.

alternatively we can have prometheus alerts setup when brick is down in host.

Reply:
Let us wait for the decision about the process metrics for a brick above. May be this will be a duplicate metric if we agree that we will show PID->brick mapping.